source/features/sticky-notifications-actions.css

```
@@ -0,0 +1,10 @@
+.js-notifications-container .Box:has(.js-select-all-text) {
```

```suggestion
.notifications-list > .Box-header {
```

I have been using this for a long time, should not need `:has`